tuf/client/updater.py

```diff
@@ -1845,7 +1867,93 @@ def _update_metadata(self, metadata_role, upperbound_filelength, version=None):
     logger.debug('Updated ' + repr(current_filepath) + '.')
     self.metadata['previous'][metadata_role] = current_metadata_object
     self.metadata['current'][metadata_role] = updated_metadata_object
-    self._update_versioninfo(metadata_filename)
+    if not snapshot_merkle:
+      self._update_versioninfo(metadata_filename)
+
+
+
+
+
+  def _verify_merkle_path(self, metadata_role):
+    """
+    <Purpose>
+      Download the merkle path associated with metadata_role and verify the hashes.
+    <Arguments>
+      metadata_role:
+        The name of the metadata role. This should not include a file extension.
+    <Exceptions>
+      tuf.exceptions.RepositoryError:
+        If the snapshot merkle file is invalid or the verification fails
+    Returns the snapshot information about metadata role.
```

Comment:
Docstring is malformed here
```suggestion
    <Returns>
      A dictionary containing the snapshot information about metadata role,
      conforming to VERSIONINFO_SCHEMA or METADATA_FILEINFO_SCHEMA
```